flutter/ios/sentry_flutter.podspec

```diff
@@ -13,11 +13,11 @@ Sentry SDK for Flutter with support to native through sentry-cocoa.
   s.source_files     = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Sentry/HybridSDK', '8.0.0-rc.1'
```

Comment:
We should remove the duplicated dependency line here, even though its the same now.

```
s.dependency 'Sentry/HybridSDK', '8.0.0-rc.1'
s.dependency 'Sentry/HybridSDK', '8.0.0-rc.1'
```